ddp-workspace/projects/ddp-dsm-ui/src/app/dashboard-statistics/components/cards-table/counts-table.component.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewChild
+} from '@angular/core';
+import {CountsModel} from '../../models/Counts.model';
+import {MatTableDataSource} from '@angular/material/table';
+import {MatSort} from '@angular/material/sort';
+
+@Component({
+  selector: 'app-cards-table',
```

Comment:
Could we change the selector to `app-counts-table` having cards table here is a bit confusing. maybe also renaming the component folder from `cards-table` to `counts-table`

Reply:
Fixed